integr8ly/grafana-operator
# Patch nil pointer in service name configuration and fix derived Fields

## Description

- change derived fields to list
- generate k8s and crds 
- fix nil pointer errors in service name configuration logic on upgrades
## Relevant issues/tickets


## Type of change




## Checklist


## Verification steps

- deploy the operator as per documentation
- deploy datasources with `oc/kubectl apply -f deploy/examples/datasources/ -n <namespace>`
- the operator and grafana should not hang when trying to handle derived fields from `simpleJsonWithDerivedFields.yaml`
